apiary.apib

```diff
@@ -3374,6 +3374,12 @@ Add a draft work order.
                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
                + quantity: 3 (number, required)
        + location (Address, optional)
+       + work_hours (array, optional)
+           + (object)
+               + user_id: `4ec55a8c-2d80-472a-a9c2-5ff5ee7eac6a` (string, required)
+               + started_at: `2019-02-27T10:01:49+00:00` (string, required)
+               + duration: 3600 (number, required) - In seconds
```

Comment:
Will we change this to match TimeTracking format? 
`started_at` & `duration` OR  `started_at` & `ended_at`

Reply:
No. This was in the initial proposal but we choose to not implement this immediately. The mobile team only uses the duration.